ci-operator/step-registry/redhat-appstudio/clean-external-resources/redhat-appstudio-clean-external-resources-commands.sh

```diff
@@ -47,7 +47,8 @@ make clean-github-webhooks || CLEAN_WEBHOOK_STATUS=$?
 make clean-quay-repos-and-robots || CLEAN_QUAY_REPOS_AND_ROBOTS_STATUS=$?
 make clean-quay-tags || CLEAN_QUAY_TAGS_STATUS=$?
 make clean-private-repos || CLEAN_PRIVATE_REPO_STATUS=$?
+make clean-registered-servers || CLEAN_PRIVATE_REPO_STATUS=$?
```

Comment:
Will it be good to capture `make clean-registered-servers` status to another variable like `CLEAN_REGISTERED_SERVERS_STATUS` and use it in the next step to check if it is `-ne 0`?

Reply:
ah, my bad, thank you for catching this issue. fixed.